aeronet/dataset/raster/collection.py

```diff
@@ -153,16 +153,16 @@ def reproject(self, dst_crs, directory=None, interpolation='nearest'):
         r_bands = []
         for band in self:
             fp = os.path.join(directory, band.name + '.tif') if directory else None
-            r_band = band.reproject(dst_crs, fp=fp, interpolation=interpolation)
+            r_band = band.reproject(dst_crs, dst_res, fp=fp, interpolation=interpolation)
```

Comment:
it is better to call optional arguments by name, not positional, in case we will change signature